models/alkhalaf2024/metadata.yaml

```diff
@@ -0,0 +1,31 @@
+title: Ontology to Explain SARS-CoV-2 Variants' Effects
+acronym: OntoEffect
+issued: 2023
+modified: 
+contributor:
+ - https://dblp.org/pid/292/3745
+ - https://dblp.org/pid/25/929-2
+ - https://dblp.org/pid/201/2998
+keyword:
+ - genetic mutations
+ - viral fitness
+ - host-virus interaction
+theme: Class Q - Science 
+editorialNote: 
+ontologyType:
+ - domain
+language: en
+designedForTask:
+ - conceptual clarification
+ - interoperability
+ - ontological analysis
+context:
+ - research
+source:
+ - https://tinyurl.com/5n8c7k8t
+representationStyle:
+ - ontouml
+landingPage:
+ - https://hdl.handle.net/11311/1256092
```

Comment:
The landing page field should not point to a paper. The source field should be used for that.

Reply:
Done.
source substituted to https://dx.doi.org/10.5220/0000178400003598
landingPage removed.